### PR TITLE
Make Checkout V2 the default

### PR DESCRIPTION
**Pull request by NicolasDorier**
The PR:

* New stores are V2
* Preset for ecommerce and retail both use V2
* Old stores which never have set checkoutType will keep using V1 so we don't change their settings.
* In checkout experience, rename the checkbox from `Use new checkout` to `Use classic checkout`

Ping @dennisreimann

### CODE REVIEW

File: BTCPayServer/Views/UIStores/CheckoutAppearance.cshtml [hunk 1]
```diff
@@ -94,21 +94,12 @@
 			</h3>
             
             <div class="d-flex align-items-center mb-3">
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIStores/CheckoutAppearance.cshtml:96]:
```suggestion
            <div class="d-flex align-items-center mb-4">
```

**Comment by dstrukt** [comment]
Hell. Yes.

**Comment by Ryleeryatt** [comment]
How so I get paied

**Comment by petzsch** [comment]
> How so I get paied

https://blog.btcpayserver.org/bitcoin-bonus-scam/

**Pull request merged**